lib/aruba/cucumber/command.rb

```diff
@@ -28,6 +29,17 @@
   run_simple(cmd, :fail_on_error => true, :exit_timeout => secs && secs.to_i)
 end
 
+When(/^I run the following commands(?: (?:with|in) `([^`]+)`)?:$/) do |shell, commands|
```

Comment:
Old
~~~
When(/^I run the following commands(?: (?:with|in) `([^`]+)`)?:$/) 
~~~

Suggestion for new
~~~
When(/^I run the following (?:commands|script)(?: (?:with|in) `([^`]+)`)?:$/)
~~~